tox.ini

```
@@ -122,47 +122,9 @@ commands =
 pypi_filter =
 extras =
 deps =
-conda_deps =
-    asdf
-    asdf-astropy
-    astropy
-    beautifulsoup4
-    cdflib
-    conda
-    dask
-    drms
-    extension-helpers
-    glymur
-    h5netcdf
-    hypothesis
-    jinja2
-    jplephem
-    libopenblas>=0.3.12
-    lxml
-    matplotlib
-    mpl_animators
-    numpy
-    opencv
-    openjpeg
-    pandas
-    parfive
-    pillow
-    py-opencv
-    pytest
-    pytest-astropy
-    pytest-cov
-    pytest-mock
-    pytest-mpl
-    pytest-xdist
-    reproject
-    scikit-image
-    scipy
-    sphinx
-    sqlalchemy
-    towncrier
-    zeep
-conda_channels = conda-forge
-install_command = pip install --no-deps {opts} {packages}
+conda_env = sunpy-dev-env.yml
+install_command = pip install --no-deps --no-build-isolation {opts} {packages}
+whitelist_externals = conda
```

```suggestion
allowlist_externals = conda
```
This needs to be changed near the top of the file as well.